src/main/java/uk/gov/ons/census/casesvc/logging/EventLogger.java

```diff
@@ -34,59 +37,65 @@ public void logEvent(
     event.setSource(EVENT_SOURCE);
     event.setChannel(EVENT_CHANNEL);
 
-    logEvent(uacQidLink, eventDescription, eventType, convertObjectToJson(payload), event, null);
+    logEvent(uacQidLink, eventDescription, eventType, convertObjectToJson(payload), event);
   }
 
   public void logReceiptEvent(
       UacQidLink uacQidLink,
       String eventDescription,
       EventType eventType,
       ReceiptDTO payload,
-      EventDTO event,
-      OffsetDateTime eventMetaDataDateTime) {
-
-    logEvent(
-        uacQidLink,
-        eventDescription,
-        eventType,
-        convertObjectToJson(payload),
-        event,
-        eventMetaDataDateTime);
+      EventDTO event) {
+
+    logEvent(uacQidLink, eventDescription, eventType, convertObjectToJson(payload), event);
   }
 
   public void logRefusalEvent(
       UacQidLink uacQidLink,
       String eventDescription,
       EventType eventType,
       RefusalDTO payload,
-      EventDTO event,
-      OffsetDateTime eventMetaDataDateTime) {
-
-    logEvent(
-        uacQidLink,
-        eventDescription,
-        eventType,
-        convertObjectToJson(payload),
-        event,
-        eventMetaDataDateTime);
+      EventDTO event) {
+
+    logEvent(uacQidLink, eventDescription, eventType, convertObjectToJson(payload), event);
+  }
+
+  public void logFulfilmentRequestedEvent(
```

Comment:
Think this method needs a direct test as well, similar to [this one for receipting](https://github.com/ONSdigital/census-rm-case-processor/blob/0a08da43334c87ed1795734a21bf51ec235140a2/src/test/java/uk/gov/ons/census/casesvc/logging/EventLoggerTest.java#L89)?

Reply:
Test added 👍